lib/v2/screens/app/interactor/usecases/guardians_notification_use_case.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:seeds/providers/services/firebase/firebase_database_service.dart';
+import 'package:seeds/v2/datasource/local/settings_storage.dart';
+
+class GuardiansNotificationUseCase {
+  Stream<bool?> get hasGuardianNotificationPending {
```

Comment:
we dont need this to be `Stream<bool?>` we can make this `Stream<bool>` if we default to false.